Public/Article.html

```diff
@@ -185,20 +205,20 @@ <h2>Comments</h2>
                 const commentForm = document.getElementById('comment-form');    // Comment form submission
                 commentForm.addEventListener('submit', function (event) {
                     event.preventDefault();     // Prevent the default form submission behavior
-
+                
                     const commentTextarea = document.getElementById('comment');
                     const newCommentBody = commentTextarea.value;
 
                     const newComment = {    // Add the new comment to the comments list
                         comment_body: newCommentBody,
-                        comment_username: 'CurrentUser', // You can replace this with the actual username or user data
+                        comment_username: CurrentUser[0].username, // Current user is returned from the getUserName function 
```

Comment:
CurrentUser should only have one user in it. CurrentUser[0] will be undefined. I think it should be something like 'CurrentUser.username'.